src/main/resources/assets/jei/lang/de_DE.lang

```diff
@@ -1,64 +1,88 @@
 # Tooltips
-jei.tooltip.config=Konfiguration
+jei.tooltip.config=JEI-Konfiguration
+jei.tooltip.show.recipes=Zeige Rezepte
 jei.tooltip.show.all.recipes=Zeige alle Rezepte
 jei.tooltip.delete.item=Klicke zum Löschen
 jei.tooltip.liquid.amount.with.capacity=%,d / %,d mB
 jei.tooltip.liquid.amount=%,d mB
+jei.tooltip.transfer=Bewege Gegenstände
+jei.tooltip.recipe.ore.dict=Akzeptiert: %s
+jei.tooltip.item.colors=Farben: %s
+jei.tooltip.shapeless.recipe=Formloses Rezept
+jei.tooltip.cheat.mode=Cheat-Modus
 
 # Error Tooltips
-jei.tooltip.error.recipe.transfer.missing=Fehlende Items
-jei.tooltip.error.recipe.transfer.inventory.full=Inventar ist voll
+jei.tooltip.error.recipe.transfer.missing=Fehlende Gegenstände
```

Comment:
Well... No, I can't go further. What really confuses here is what is resulting from the fact that programmers don't give enough space for other languages. Being constistent in terms of beginning to not translate a word *everywhere* breaks the logic of a consistent translation completely. Instead, giving enough space would be the best solution in order to make it possible to translate correctly. If the programmer is not willing to do so, than it means he doesn't care about other languages and therefore doesn't want a good localization.
So, please go ahead and make Mekanism and similar mods translation friendly instead of violating a language because of trivialities. It's not impossible.